src/main/java/cn/nukkit/entity/EntityEnderCrystal.java

```diff
@@ -0,0 +1,83 @@
+package cn.nukkit.entity;
+
+import cn.nukkit.Player;
+import cn.nukkit.event.entity.ExplosionPrimeEvent;
+import cn.nukkit.level.Explosion;
+import cn.nukkit.level.format.FullChunk;
+import cn.nukkit.math.Vector3;
+import cn.nukkit.nbt.tag.CompoundTag;
+import cn.nukkit.network.protocol.AddEntityPacket;
+
+public class EntityEnderCrystal extends EntityLiving implements EntityExplosive {
```

Comment:
@kvetinac97 Be serious about everything you're talking. If you are always asking without contributing like this, we will block you outside from this project.